Mlem/App/Views/Pages/Instance/InstanceView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  InstanceView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 08/07/2024.
+//
+
+import LemmyMarkdownUI
+import MlemMiddleware
+import SwiftUI
+
+struct InstanceView: View {
+    enum Tab: String, CaseIterable, Identifiable {
+        case about, administration, details, uptime, safety
+        
+        var label: String {
+            switch self {
+            case .safety:
+                "Trust & Safety"
+            default:
+                rawValue.capitalized
+            }
+        }
+        
+        var id: Self { self }
+    }
+    
+    @Environment(Palette.self) var palette
+    @Environment(AppState.self) var appState
+    @Environment(\.colorScheme) var colorScheme
+    
+    // This is fetched from the instance itself, not from the logged-in account.
+    @State var instance: any InstanceStubProviding
+    @State var upgradeState: LoadingState = .idle
+    
+    @State var selectedTab: Tab = .about
+    @State var isAtTop: Bool = true
+    
+    // If != nil, blocking is in progress
+    @State var isBlocking: UUID?
+    
+    init(instance: any InstanceStubProviding) {
+        self.instance = instance
```

Comment:
Fails build on Xcode 
```suggestion
        self._instance = .init(wrappedValue: instance)
```